source/agora/consensus/Ledger.d

```diff
@@ -924,12 +842,7 @@ public class ValidatingLedger : NodeLedger
                 data.enrolls ~= enroll;
             }
         }
-
-        if (auto reason = this.externalize(data, sign_all))
-        {
-            assert(0, format!"Failure in unit test. Block %s should have been externalized: %s"(
-                       this.height() + 1, reason));
-        }
```

Comment:
This change seems unrelated, and I don't think we want it. `forceCreateBlock` should not fail (it's a test-only utility).

Reply:
Reverted